libs/application/templates/estate/src/lib/constants.ts

```diff
@@ -33,3 +33,30 @@ export const EstateTypes = {
   permitToPostponeEstateDivision: 'Búsetuleyfi',
   divisionOfEstateByHeirs: 'Einkaskipti',
 }
+
+export const EstateApplicationOptions =
+  process.env.NODE_ENV !== 'production'
+    ? [
+        {
+          value: EstateTypes.divisionOfEstate,
+          label: EstateTypes.divisionOfEstate,
+        },
+        {
+          value: EstateTypes.estateWithoutAssets,
+          label: EstateTypes.estateWithoutAssets,
+        },
+        {
+          value: EstateTypes.permitToPostponeEstateDivision,
+          label: EstateTypes.permitToPostponeEstateDivision,
+        },
+        {
+          value: EstateTypes.divisionOfEstateByHeirs,
+          label: EstateTypes.divisionOfEstateByHeirs,
+        },
+      ]
+    : [
+        {
+          value: EstateTypes.permitToPostponeEstateDivision,
+          label: EstateTypes.permitToPostponeEstateDivision,
+        },
+      ]
```

Comment:
I'm ok with this but it will slow down the process of releasing.
It may be considered bad form to hotfix these switches but I am not sure :)

Reply:
Although, I doubt a quadruple feature flag is the answer either :laughing:

Reply:
I was pointed at much better solution actually, stay tuned!